.github/workflows/build.yml

```diff
@@ -25,7 +25,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        go: [1.15, 1.16, 1.17, "1.18.0-rc1"]
+        go: [1.15, 1.16, 1.17, 1.18]
```

Comment:
Let's drop 1.15 ?

Reply:
Should we bump the `go.mod` version to Go 1.16 then as well?

Reply:
Or maybe release a new version first, especially with #175 and #139?